vertical-pod-autoscaler/pkg/admission-controller/main.go

```diff
@@ -41,8 +42,11 @@ var (
 		tlsPrivateKey: flag.String("tls-private-key", "/etc/tls-certs/serverKey.pem", "Path to server certificate key PEM file."),
 	}
 
-	address   = flag.String("address", ":8944", "The address to expose Prometheus metrics.")
-	namespace = os.Getenv("NAMESPACE")
+	address         = flag.String("address", ":8944", "The address to expose Prometheus metrics.")
+	namespace       = os.Getenv("NAMESPACE")
+	webhookHostName = flag.String("webhookHostName", "", "path for admission controller.Blank for default")
+	serverPort      = flag.String("Server Port", "", "Server Port for Webhook")
+	registerByURL   = flag.Bool("registerByURL", false, "True, if admission webhook is to be register on specific URL")
```

Comment:
Can you change the description to "If set to true, admission webhook will be registered by URL (:) instead of by service name."